### ISSUE TITLE
Hyprland: Workspace renaming breaks 'window' updates

### ISSUE THREAD

**Issue by BinaryInk** [Alexays/Waybar#3759]
# Problem
Upon renaming a workspace, the displayed icons provided by `"format": "{windows}"

# Reproduction
1. Open a window in a given workspace.
2. Rename the workspace via `hyprctl dispatch renameworkspace "$activeWorkspaceId" "$newName" -i 0`
3. Open another window, and a new window icon will not appear as expected.

# Notes
- Upon resetting the name back to its original state, new windows will appear as expected when opened.
- Other un-renamed workspaces are unaffected
- This is occurs with specially named workspaces as well (e.g., not the same as the workspace ID)
- Closing waybar and re-opening will cause the icon to appear in the renamed workspace, but the problem still persists with newly created windows in the renamed workspace.

# Conclusion
In short: it only occurs when the name is different from when the Hyprland session was started. 

As far as I can tell, this is not related to #2710 -- if only because this is newly observed behavior occurring within the past month. I presume it's related to a new Hyprland release, but I cannot confirm that at the moment.
